File: unit-2/sprint-1/Task Management Layout/app/src/main/java/com/example/tsm/MainActivity.java
package com.example.tsm;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button mbutton;
    private Button mbtnlogin;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        intidview();
        mbutton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this,CreateAccountActivity.class);
                startActivity(intent);
            }
        });

        mbtnlogin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this,LoginActivity.class);
                startActivity(intent);
            }
        });
    }

    private void intidview() {
        mbtnlogin = findViewById(R.id.btnlogin);
        mbutton = findViewById(R.id.button);
    }
}

File: unit-4/sprint-2/Simple Expense Manager/Expense Manager App/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.google.android.material.floatingactionbutton.FloatingActionButton
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val fragmentAdapter = ConnectFragmentAdapter(supportFragmentManager)
        fragmentAdapter.addFragment(IncomeFragment(),"Income")
        fragmentAdapter.addFragment(ExpenseFragment(),"Expense")
        fragmentAdapter.addFragment(BalanceFragment(),"Balance")

        viewPager.adapter= fragmentAdapter
        tabLayout.setupWithViewPager(viewPager)
        findViewById<FloatingActionButton>(R.id.fab).setOnClickListener { view ->
            intent = Intent(this,InsertDataActivity::class.java)
            startActivity(intent)
        }
    }
}